MonoGame.Framework/Platform/Graphics/OpenGL.iOS.cs

```diff
@@ -35,7 +35,13 @@ public class GraphicsContext : IGraphicsContext
     {
         public GraphicsContext ()
         {
-            Context = new EAGLContext (EAGLRenderingAPI.OpenGLES2);
+            nuint major, minor;
+            EAGLContext.EAGLGetVersion(out major, out minor);
+
+            if (major >= 3)
+                Context = new EAGLContext (EAGLRenderingAPI.OpenGLES3);
+            else
+                Context = new EAGLContext (EAGLRenderingAPI.OpenGLES2);
```

Comment:
I just added the fallback check. Could you be sure this is capable? @tomspilman @Jjagg